CHANGELOG.md

```diff
@@ -15,6 +15,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ### Fixed
 
+- Enforce UTF-8 encoding in the CSV download via a BOM (byte order mark)
```

Comment:
I don't think we are enforcing UTF-8 - but we should! It probably happens inside `OpeningProjectsCsvExporter`: `CSV.generate`